opus/application/static_media/js/browse.js

```diff
@@ -1541,12 +1647,10 @@ var o_browse = {
                         obsNum = (lastObs !== undefined ? lastObs + 1 : obsNum);
                         customizedLimitNum = o_browse.getLimit(view);
 
-                        // If it's in table view, we will update the obsNum in InfiniteScroll instances with the
-                        // startObs values right before infiniteScroll load event is triggered. This will be used
-                        // to properly set the scrollbar and slider location in table view after new data is loaded.
-                        let scrollbarObsNum = (o_browse.isGalleryView() ? (obsNum - o_browse.getLimit(view) -
-                                               opus.getViewNamespace(view).galleryBoundingRect.x) :
-                                               opus.prefs[startObsLabel]);
+                        // We will update the obsNum in InfiniteScroll instances with the startObs values
+                        // right before infiniteScroll load event is triggered. This will be used to properly
+                        // set the scrollbar and slider location after new data is loaded.
+                        let scrollbarObsNum = opus.prefs[startObsLabel];
                         scrollbarObsNum = Math.max(scrollbarObsNum, 1);
```

Comment:
can scrollbarObsNum ever be less than 1?

Reply:
Because scrollbarObsNum is used to stored the top item in table view (or one of the top row items in gallery view), the minimum number will be 1.

Reply:
I think Debby's point was that if opus.prefs[startObsLabel] can never be less than 1, then there's no point in doing the max() and it can be removed.